src/main/interface/cli.c

```diff
@@ -382,7 +393,7 @@ static void cliPrintErrorLinef(const char *format, ...)
 }
 
 
-static void printValuePointer(const clivalue_t *var, const void *valuePointer, bool full)
+static void printValuePointer(const clivalue_t *var, const void *valuePointer, bool full, dumpValue_e type)
```

Comment:
Not happy with having to add a parameter for one specific feature into a generic function - we should try to solve this in a way that leaves `printValuePointer()` a generic function.

Reply:
I agree, I don't like this solution, but could not come up with a cleaner/better one - yet.

Reply:
What's wrong with normal value handling?

Reply:
@pkruger What I suggested in https://github.com/betaflight/betaflight/pull/6714#issuecomment-419696968 is cleaner and removes the need for the custom CLI code.